examples/confluent_cloud.py

```diff
@@ -48,42 +48,76 @@
 
 import uuid
 
-from confluent_kafka import Producer, Consumer
+from confluent_kafka import Producer, Consumer, KafkaError, KafkaException
 
 
+def error_cb(err):
+    """ The error callback is used for generic client errors. These
+        errors are generally to be considered informational as the client will
+        automatically try to recover from all errors, and no extra action
```

Comment:
> will automatically try to recover from all errors

what about fatal errors? probably should mention those, especially as I assume we will make idempotent producer the default before too long like Java, at which point they will apply to this example.

Reply:
I think we'll want to take a bigger grab and introduce the error attributes, treating _AUTHENTICATION as a fatal error, rather than having certain error codes and fatal attribute.

Reply:
oh, we don't have these in python yet. ok.

don't think auth is fatal in all cases.